upload_sig_storage_driver.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+VERSION=${1:?"Missing version number argument (arg 1)"}
+NEXUS_USER=${2:?"Missing Nexus username argument (arg 2)"}
+PRODUCT=${3:?"Missing product argument (arg 3)"}
+REGISTRY="registry.k8s.io/sig-storage"
+STACKABLE_REGISTRY="docker.stackable.tech/k8s/sig-storage"
+
+ARCHITECTURES=(
+    amd64
+    arm64
+)
+
+function pull_retag_and_push_images() {
+    for arch in "${ARCHITECTURES[@]}"; do
+        docker pull "${REGISTRY}/${PRODUCT}:v${VERSION}" --platform linux/${arch}
```

Comment:
I'm sorry, I'm too stupid to select multiple lines to make a suggestion :(

```
        echo "Pulling ${REGISTRY}/${PRODUCT}:v${VERSION} for ${arch}"
        docker pull "${REGISTRY}/${PRODUCT}:v${VERSION}" --platform "linux/${arch}"
        
        echo "Retagging image for ${arch}"
        docker tag "${REGISTRY}/${PRODUCT}:v${VERSION}" "${STACKABLE_REGISTRY}/${PRODUCT}:v${VERSION}-${arch}"
        
        echo "Pushing retagged image for ${arch}"
        docker push "${STACKABLE_REGISTRY}/${PRODUCT}:v${VERSION}-${arch}"
```

Basically adds comments and fixes shellcheck warning.

Reply:
Done https://github.com/stackabletech/docker-images/pull/668/commits/3082d37d8ce1bb1dd096628fb51e152c1823fffd